impl/src/main/java/org/jboss/cdi/tck/tests/context/dependent/Missile.java

```diff
@@ -16,9 +16,7 @@
  */
 package org.jboss.cdi.tck.tests.context.dependent;
 
-import jakarta.interceptor.Interceptors;
-
-@Interceptors(MissileInterceptor.class)
+@MissleBinding
```

Comment:
Doesn't have a bean defining annotation. But it looks like the `Missile` class (and the interceptor) are not used in this test at all?